docs/source/processing_at_scale/processing_with_aws.rst

```diff
@@ -0,0 +1,351 @@
+
+Here's what our's looks like:
+
+.. figure:: images/aws-tutorial-figure3.png
+   :align: center
+
+Now open the file named `register-merge-job.json`. This file describes a batch job that will create an ec2 instance using the docker container `spacetx/merge-batch-job` that merges together all your processed results into
+one `DecodedIntensityTable`. Again replace the string "ADD ARN" with the aws ARN of your batch processing role. Our file looks like this:
+
+::
+
+    {
+      "jobDefinitionName": "merge-job",
+      "type": "container",
+      "containerProperties": {
+        "jobRoleArn": "arn:aws:iam::422553907334:role/spacetx-batch",
+        "image": "spacetx/merge-batch-job",
+        "vcpus": 1,
+        "memory": 2500
+      }
+    }
+
+Then from the directory where this file lives run the following command:
+
+``aws batch submit-job --cli-input-json file://register-merge-job.json``
+
+Again, check that your job has been successfully registered from the job console, our two jobs are ready to go!
+
+.. figure:: images/aws-tutorial-figure4.png
+   :align: center
+
+
+Run your Batch Jobs
+-------------------
+
+Now that we've set everything up it's time to run our jobs! The script `starfish-workflow.py` will handle submitting the process-fov array job
+then the merge job with a dependency on the first job to finish. All you'll need to do is run the script with a few parameters:
+
+::
+
+    --experiment-url: The url to your experiment.json file. Our is "https://s3.amazonaws.com/aws-processing-example/iss-spacetx-formatted/experiment.json"
+
+    --num-fovs: The number of fields of view in the experiment. We have 15
+
+    --recipe-location: The path to your recipe file in s3. Ours is "s3://aws-processing-example/aws-processing-example/iss-spacetx-formatted/recipe.py"
+
+    --results-location: The s3 bucket to copy the results from the job to. Ours is "s3://aws-processing-example/iss-spacetx-formatted-results/"
+
+    --job-queue: The name of your job queue to run your jobs. Ours is "first-run-job-queue"
+
+
+Now we run our script:
+
+::
+
+    $ python3 starfish-workflow.py \
+    >     --experiment-url "https://s3.amazonaws.com/aws-processing-example/iss-spacetx-formatted/experiment.json" \
+    >     --num-fovs 15 \
+    >     --recipe-location "s3://aws-processing-example/aws-processing-example/iss-spacetx-formatted/recipe.py" \
+    >     --results-bucket "s3://aws-processing-example/iss-spacetx-formatted-results/" \
+    >     --job-queue "first-run-job-queue"
+    Process fovs array job 39a13edd-8cca-4e7e-9379-aa3cf757c72e successfully submitted.
+    Merge results job ac5d49f5-a12e-4176-96e2-f697c6cf0a12 successfully submitted.
+
+To monitor the status of both jobs navigate to the `AWS Batch Dashboard <https://console.aws.amazon.com/batch/home?region=us-east-1#/dashboard>`_. You should see 2
```

Comment:
The problem with this is that this encodes the region in the URL.  Is there a way to get a region-agnostic dashboard?

Reply:
I think https://console.aws.amazon.com/batch/home will automatically navigate to the dashboard, so I'll jsut change it to that